Yoast/javascript
# Adds Indonesian transition words assessment

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Implements the transition words assessment for Indonesian.
* [Yoast SEO Free] Adds the assessment for transition words in Indonesian.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Add more specs to `findTransitionWordsSpec`. Use sentences that contain a transition word (either single word or multiple word), a two-part transition word, or no transition word at all. 
* Run `yarn test` and make sure that all tests pass.

* In your WordPress testing environment, set the language to Indonesian.
* Open a new post or page.
* Write Indonesian sentences that:
    * contain a single-word transition word, like `selanjutnya`, `ringkasnya`, `karenanya` .
    * E.g. `Ringkasnya, ia telah menyelesaikan pendidikannya dengan gemilang.`, `Selanjutnya, ia berencana untuk tinggal di Australia.`, `Karenanya, ia telah menjadi pengusaha sukses sekarang.`
    * contain a multi-word transition word, like `bagaimanapun juga`, `dalam hal itu`, `pada dasarnya`.
    * E.g. `Bagaimanapun juga, ibunya adalah orang yang paling berjasa dalam hidupnya.`, `Dalam hal itu, ia lebih memilih untuk tidak ikut campur.`, `Pada dasarnya, manusia adalah makhluk sosial.`
    * contain a two-part transition word, like `demikian, sehingga`, `bukan hanya, melainkan juga`, `biarpun, namun`.
    * E.g. `Perempuan itu bukan hanya cerdas dalam hal intelektualnya, melainkan juga dalam hal emosionalnya.`, `Biarpun raganya terasa berat, namun ia tetap semangat membanting tulang demi keluarganya.`
* Hit the mark `transition words` assessment button to check if the sentences are correctly marked.
## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-484
